tests/components/enocean/test_binary_sensor.py

```diff
@@ -0,0 +1,66 @@
+"""Test EnOcean light implementation."""
+import unittest
+from unittest.mock import MagicMock, patch
+
+from enocean.protocol.packet import RadioPacket
+
+from homeassistant.components import enocean as enocean_component
+from homeassistant.components.enocean import binary_sensor
+from homeassistant.const import STATE_OFF
+
+from tests.common import get_test_home_assistant
+
+
+class TestEnOceanBinarySensor(unittest.TestCase):
+    """Test cases for the EnOcean binary sensor."""
+
+    def setUp(self):
+        """Initialize values for this test case class."""
+        enocean_component.ENOCEAN_DONGLE = MagicMock()
+        self.device = binary_sensor.EnOceanBinarySensor(
```

Comment:
We should set up the corresponding component, via the core, which sets up the platform which adds an entity. Use `setup_component` or `async_setup_component` imported from setup.py.

Reply:
Tests removed